packages/now-cli/test/dev-server.unit.js

```diff
@@ -138,16 +146,23 @@ test(
 test(
   '[DevServer] Allow `cache-control` to be overwritten',
   testFixture('now-dev-headers', async (t, server) => {
-    const res = await get(
+    const res = await fetch(
       `${server.address}/?name=cache-control&value=immutable`
     );
-    t.is(res.headers['cache-control'], 'immutable');
+    t.is(res.headers.get('cache-control'), 'immutable');
   })
 );
 
 test(
   '[DevServer] Sends `etag` header for static files',
   testFixture('now-dev-headers', async (t, server) => {
+    if (process.platform === 'win32') {
+      console.log(
+        'Skipping "etag" test on windows since it yields a different result.'
+      );
+      t.is(true, true);
+      return;
+    }
     const res = await fetch(`${server.address}/foo.txt`);
     t.is(res.headers.get('etag'), '"d263af8ab880c0b97eb6c5c125b5d44f9e5addd9"');
```

Comment:
I think changing this test to this would be better than skipping:

```js
     t.is(/^"[a-f0-9]{40}"$/.test(res.headers.get('etag')), true);
```

Reply:
But it seems like testing the actual etag value is important. I'm guessing its a bug in Windows that we need to fix.

Reply:
I updated `.gitattributes` so that tests are checked out with `\n`, even on windows.